apps/hestia/src/components/ui/ConnectWalletInteraction/connect.tsx

```diff
@@ -135,21 +137,10 @@ const CardsCompontent = ({ onClose, onNext }: InteractableProps) => {
 
   return (
     <Fragment>
-      <Interactable.Card pageName="Authorization">
-        <Authorization
-          onAction={async () =>
-            await connectExtensionAccounts(selectedExtension?.id as string)
-          }
-          extensionIcon={selectedExtension?.id as string}
-          extensionName={selectedExtension?.title}
-          onActionCallback={() => setPage("ConnectFundingWallets")}
-          onClose={onReset}
-        />
-      </Interactable.Card>
       <Interactable.Card pageName="ConnectFundingWallets">
         <ExtensionAccounts
           extensionAccounts={walletsFiltered}
-          loading={!!mainProxiesLoading || !!walletLoading}
+          loading={false} // TEMP
```

Comment:
I cannot remove this for now, I will correct it in Polkadex/ux first

Reply:
I basically added to show loading, until balance is being fetched. If we hardcode it to false, it may bring unwanted issues. Like, it may show 0 balance even if it is being fetched.